### PR TITLE
Added i18n support

### PR DESCRIPTION
**Pull request by jbristowe**

### CODE REVIEW

File: app/Frame.js [hunk 1]
```diff
@@ -3,6 +3,110 @@ import { AppContext } from "./context/AppContext";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 import { useMediaPredicate } from "react-media-hook";
 
+/* CLDR Data */
```
Comment (httpJunkie) [inline app/Frame.js]: Add to barrel file so that there is only one import here

File: app/Frame.js [hunk 2]
```diff
@@ -3,6 +3,110 @@ import { AppContext } from "./context/AppContext";
...
+import usLocalCurrency from 'cldr-numbers-full/main/en/currencies.json';
+import usCaGregorian from 'cldr-dates-full/main/en/ca-gregorian.json';
+import usDateFields from'cldr-dates-full/main/en/dateFields.json';
+
+import zhNumbers from 'cldr-numbers-full/main/zh/numbers.json';
+import zhLocalCurrency from 'cldr-numbers-full/main/zh/currencies.json';
+import zhCaGregorian from 'cldr-dates-full/main/zh/ca-gregorian.json';
+import zhDateFields from'cldr-dates-full/main/zh/dateFields.json';
+
+import { IntlProvider, load, loadMessages, LocalizationProvider } from '@progress/kendo-react-intl';
```
Comment (httpJunkie) [inline app/Frame.js]: This import should not be in the file you create to have all of the imports (I dont think). Can still be in the Frame,js on it's own? Or does it make sense to extract it as well? I'll let you make that choice.

File: app/Frame.js [hunk 3]
```diff
@@ -3,6 +3,110 @@ import { AppContext } from "./context/AppContext";
...
+    usLocalCurrency,
+    usCaGregorian,
+    usDateFields,
+    zhNumbers,
+    zhLocalCurrency,
+    zhCaGregorian,
+    zhDateFields
+);
+
+import deMessages from '../lib/kendo-react-messages/messages/de-DE/de-DE.json';
```
Comment (httpJunkie) [inline app/Frame.js]: From here down should be the second gang of imports that are extracted to another file.

File: app/Frame.js (hunk 1 above)
Comment (jbristowe) [reply app/Frame.js]: Done.

**Comment by jbristowe** [comment]
Added missing files.

**Pull request merged**